OpenRA.Game/Translation.cs

```diff
@@ -53,63 +63,65 @@ static IEnumerable<MessageContext> GetMessageContext(string language, string[] t
 				var stream = fileSystem.Open(path);
 				using (var reader = new StreamReader(stream))
 				{
-					var options = new MessageContextOptions { UseIsolating = false };
-					var messageContext = new MessageContext(language, options);
-					var errors = messageContext.AddMessages(reader);
+					var parser = new LinguiniParser(reader);
+					var resource = parser.Parse();
+					var errors = resource.Errors;
 					foreach (var error in errors)
 						Log.Write("debug", error.ToString());
 
-					yield return messageContext;
+					bundle.AddResourceOverriding(resource);
 				}
 			}
 		}
 
-		public string GetFormattedMessage(string key, IDictionary<string, object> args = null, string attribute = null)
+		public string GetString(string key, IDictionary<string, object> arguments = null)
 		{
-			if (key == null)
-				return "";
-
-			foreach (var messageContext in messageContexts)
+			if (!TryGetString(key, out var message, arguments))
 			{
-				var message = messageContext.GetMessage(key);
-				if (message != null)
-				{
-					if (string.IsNullOrEmpty(attribute))
-						return messageContext.Format(message, args);
-					else
-						return messageContext.Format(message.Attributes[attribute], args);
-				}
+				Log.Write("debug", $"Unknown messageId {key}");
```

Comment:
Calling `GetString` with a real string instead of a translation key is a valid use case. Won't this generate a lot of unwanted log spam?